doc/changelog/07-ssreflect/18449-changelog-ssr.rst

```diff
@@ -0,0 +1,7 @@
+- **Changed:**
+  The :tacn:`have` tactic generates a proof term containing an opaque
+  constant, as it did up to PR `#15121 <https://github.com/coq/coq/pull/15121>`_
+  included in Coq 8.16.0. See the variant `have @H` to generate a let-in instead.
```

Comment:
I suggest adding "transparent" before "let-in" (quotes are not part of the added text).

Reply:
It may also be nice to add a link to https://coq.inria.fr/doc/v8.19/refman/proof-engine/ssreflect-proof-language.html?highlight=have#generating-let-in-context-entries-with-have

Reply:
Should be better now.